src/domexpression.js

```diff
@@ -49,7 +60,14 @@ var _ = Mavo.DOMExpression = $.Class({
 		if (typeof this.expression !== "string") { // Still unhandled?
 			if (this.attribute) {
 				// Some web components (e.g. AFrame) hijack getAttribute()
```

Comment:
Need to move this comment to still precede the `Element.prototype.getAttribute` call, otherwise it makes no sense. 
You could also alias it to a variable outside everything, have the comment right above that and use that in calls:

```js
// Some web components (e.g. AFrame) hijack getAttribute()
const originalGetAttribute = Element.prototype.getAttribute;
```

Reply:
Done! Thanks.